src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DeleteDescriptorsValidation.feature

```diff
@@ -15,11 +15,11 @@ Feature: Resources "Delete" Operation validations
                     }
                   """
 
-        @ignore
         Scenario: 01 Verify deleting a specific descriptor by ID
              When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
              Then it should respond with 204
 
+        # Restore with DMS-309
         @ignore
         Scenario: 02 Verify error handling when deleting a descriptor using a invalid id
```

Comment:
I discovered that we had not written code for this scenario yet, so I created a new ticket and kept the `@ignore` tag on this scenario.